packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Column/__tests__/configAggregator.spec.js

```diff
@@ -0,0 +1,100 @@
+    const node = document.createElement('div');
+    node.innerHTML = `<div class="pagebuilder-column myCssColumnClass"
+        data-content-type="column"
+        data-appearance="default"
+        data-background-images='{\\"desktop_image\\":\\"image-desktop.png\\",\\"mobile_image\\":\\"image-mobile.png\\"}'
+        data-element="main"
+        style="justify-content: flex-start;
+            display: flex;
+            flex-direction: column;
+            background-color: rgb(193, 193, 193);
+            background-position: left top;
+            background-size: cover;
+            background-repeat: no-repeat;
+            background-attachment: scroll;
+            text-align: center;
+            border-style: solid;
+            border-color: rgb(0, 0, 0);
+            border-width: 1px;
+            border-radius: 10px;
+            width: calc(33.3333% - 20px);
+            margin: 10px;
+            padding: 10px;
+            align-self: stretch;">
+    </div>`;
+
+    const aggregatedConfig = configAggregator(node.childNodes[0], {
+        appearance: 'full-height'
+    });
+
+    expect(aggregatedConfig).toEqual(
+        expect.not.objectContaining({
+            minHeight: expect.any(String)
+        })
+    );
+});
+
+test('column config is aggregated correctly retrieving all properties', () => {
+    const node = document.createElement('div');
+    node.innerHTML = `<div class="pagebuilder-column myCssColumnClass"
+        data-content-type="column"
+        data-appearance="default"
+        data-background-images='{\\"desktop_image\\":\\"image-desktop.png\\",\\"mobile_image\\":\\"image-mobile.png\\"}'
+        data-element="main"
+        style="justify-content: flex-start;
+            display: flex;
+            flex-direction: column;
+            background-color: rgb(193, 193, 193);
+            background-position: left top;
+            background-size: cover;
+            background-repeat: no-repeat;
+            background-attachment: scroll;
+            text-align: center;
+            border-style: solid;
+            border-color: rgb(0, 0, 0);
+            border-width: 1px;
+            border-radius: 10px;
+            min-height: 300px;
+            width: calc(33.3333% - 20px);
+            margin: 10px;
+            padding: 10px;
+            align-self: stretch;">
+    </div>`;
+
+    const aggregatedConfig = configAggregator(node.childNodes[0], {
+        appearance: 'full-height'
+    });
+
+    expect(aggregatedConfig).toEqual(
+        expect.objectContaining({
+            backgroundAttachment: 'scroll',
+            backgroundColor: 'rgb(193, 193, 193)',
+            backgroundPosition: 'left top',
+            backgroundRepeat: false,
+            backgroundSize: 'cover',
+            border: 'solid',
+            borderColor: 'rgb(0, 0, 0)',
+            borderRadius: '10px',
+            borderWidth: '1px',
+            cssClasses: ['pagebuilder-column', 'myCssColumnClass'],
```

Comment:
We don't currently verify advanced properties within our configAggregator tests, as these utility methods are already tested within our implementation.

See https://github.com/magento/pwa-studio/pull/1698/files#diff-f7624a8cda70ade6382454adf8f2d960 as an example, we only validate properties which are not generated from the utility methods.